tests/unit/test_variants.py

```diff
@@ -70,3 +73,112 @@ def testPhasesetZero(self):
 
     def testGenotypeHaploid(self):
         self.verifyGenotypeConversion("1", "376", [1], None)
+
+
+class TestAbstractVariantSet(unittest.TestCase):
+    """
+    Unit tests for the abstract variant set.
+    """
+    def setUp(self):
+        self._vsname = "testVariantSet"
+        self._backend = backend.AbstractBackend()
+        self._dataset = datasets.AbstractDataset(self._backend)
+        self._variantSet = variants.AbstractVariantSet(
+            self._dataset, self._vsname)
+
```

Comment:
Ah, I see. These are good tests, but the features are heavily covered elsewhere. I think we can remove them, see the comments below.